app/javascript/chat/channelDetails.jsx

```diff
@@ -167,7 +177,12 @@ class ChannelDetails extends Component {
             rel="noopener noreferrer"
             data-content={`users/${user.id}`}
           >
-            @{user.username} - {user.name}
+            @
+            {user.username}
+            {' '}
+- 
+            {' '}
```

Comment:
This is exactly the same case like before but Prettier doesn't do anything this time.